### ISSUE TITLE
Update Placement snippets and templates

### ISSUE THREAD

**Issue by TomerFi** [open-cluster-management-io/ocm-vscode-extension#29]
Placement CRD can be found [here](https://github.com/open-cluster-management-io/placement/blob/main/deploy/hub/0000_02_clusters.open-cluster-management.io_placements.crd.yaml).

Resource docs can be found [here](https://open-cluster-management.io/concepts/placement/).

Snippets are saved in our [snippets directory](https://github.com/open-cluster-management-io/ocm-vscode-extension/tree/main/snippets).

Templates are saved in our [templates directory](https://github.com/open-cluster-management-io/ocm-vscode-extension/tree/main/templates).

[VSCode Snippet Guide](https://code.visualstudio.com/api/language-extensions/snippet-guide).
[VSCode Using Snippets](https://code.visualstudio.com/docs/editor/userdefinedsnippets).

Related issue: #21 

> Templates are a part of our _ocmNewProject_ command, no vscode docs available.
> We have multiple templates, each template as its own Placement manifest.

**Comment by ilan-pinto** [comment]
why reopened

**Comment by TomerFi** [comment]
We updated only the templates. Not the snippet.

**Comment by TomerFi** [comment]
@ilan-pinto we're confusing #26 (channel) and this one (placement).
The recent PR updates the placement templates, but not the snippet:
https://github.com/open-cluster-management-io/ocm-vscode-extension/blob/main/snippets/placement.json

![image](https://user-images.githubusercontent.com/28388442/236135227-e8038310-5f2e-418a-b041-e9ce682c3423.png)

We also need to decide if we want to allow users to use old manifests. Meaning, should we add another snippet:
```json
{
	"placement-v1alpha1": {
            ...
	},
	"placement-v1beta1": {
            ...
	}
}
```
Or replace the current one:
```json
{
	"placement": {
            ...
	}
}
```

As you can see with the current object key, the original intention was to keep multiple versions.

**Comment by TomerFi** [comment]
fixed in #69
